### PR TITLE
Feature 1627 release guide

### LINKED ISSUE #1627: Update the Release Guide to add instructions for updating the Coordinated Release section on the Download page
## Describe the Task ##
Update the Release Guide to add instructions for updating the Coordinated Release section on the Download page
https://dtcenter.org/community-code/metplus/download

### Time Estimate ###
1 -4 hours

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
None

### Funding Source ###
2702691

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Linked issues**
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Read and reread the added documentation. Reviewed the added documentation under MET:
https://metplus.readthedocs.io/en/feature_1627_release_guide/Release_Guide/met_official.html#update-dtc-website
and  METplus:
https://metplus.readthedocs.io/en/feature_1627_release_guide/Release_Guide/metplus_official.html#update-dtc-website
for formatting.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the added documentation and suggest any desired modifications.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[N/A]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[20220525]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 1]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
       
     * For an official release, remove the corresponding development
       releases.
+
+  * Create or edit the "Coordinated METplus Version X.Y" software release.
+
+    * For a beta1 development release, create the "Coordinated METplus
+      Version X.Y" release section if it doesn't already exist.  Ensure the
+      "Release Type" is set to *Development*. Add links for the |projectRepo|
+      X.Y.Z beta1, the METplus "Documentation", and METplus "Existing Builds
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
      X.Y.Z beta1, the METplus "Documentation", the METplus "Existing Builds
```

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 2]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
...
+      and Docker" and under "Release Notes" add a bullet point linking to the
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
      and Docker" page, and the "Release Notes". Add to the Release Notes a link to the
```

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 3]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
...
+      |projectRepo| Release Notes with the text |projectRepo|.
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
      |projectRepo| Release Notes.
```

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 4]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
...
       
+    * For a beta>=2 development release, update the existing link and text in
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
    * For a beta2 or beyond development release, update the existing link and text in
```

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 5]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
...
+      the "Coordinated METplus Version X.Y" release section with the beta>=2
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
      the "Coordinated METplus Version X.Y" release section with the updated
```

File: docs/Release_Guide/release_steps/common/update_dtc_website.rst [hunk 6]
```diff
@@ -54,7 +54,29 @@ Update DTC Website
...
+      development release information.
+
+    * For an official release, update the existing link and text in
+      the "Coordinated METplus Version X.Y" release section with the X.Y.Z
+      information. Change the "Release Type" from *Development* to
+      *Recommended*.
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/common/update_dtc_website.rst]:
```suggestion
      *Recommended*. Consider changing the "Release Type" of previous coordinated releases from *Recommended* to *Other*.
```

Comment (jprestop) [reply docs/Release_Guide/release_steps/common/update_dtc_website.rst]: Great catch here, thank you!

**Review by JohnHalleyGotway — APPROVED** [review]
@jprestop thanks for making these updates. Overall, I approve of these changes.

I did propose several small edits to the wording to hopefully make it more clear. But this is all pretty subjective. I'll leave it to you to decide what wording you prefer.

**Pull request merged**